app/content/team/index.js

```diff
@@ -93,6 +94,15 @@ function Index() {
     setNewName(currentTeamData ? currentTeamData.name : "");
   }, [currentTeamData]);
 
+  const handleUserPress = useCallback(
+    debounce(
+      (userId) => router.push(`content/team/users/${userId}`),
+      1000,
+      true,
+    ),
+    [],
+  );
+
```

Comment:
This one could be set up to the previous ones too. it would just require its parameters be adjusted when its called.

Reply:
Addressed in #296